composer.json

```diff
@@ -7,6 +7,12 @@
         "AFL-3.0"
     ],
     "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "laminas/laminas-dependency-plugin": true,
+            "magento/composer-dependency-version-audit-plugin": true,
```

Comment:
@hostep What about a wildcard entry for Magento plugins, like `magento/*`?

Reply:
@torhoehn: it would work indeed, I just tested this out.
But since this is a security feature of composer, I think we should limit the whitelist very strictly. You never know that somebody is able to sneak in an evil composer plugin with the magento vendor namespace (not necessarily from repo.magento.com or packagist.org), it's good to keep this list as strict as possible.

(And on a personal note, I really would like to see Magento to stop creating extra composer plugins, look at the migration of composer v1 to composer v2, it was a disaster in Magento because of all the composer plugins it depends upon, this will happen again when composer v3 comes along one day, fewer plugins are better in my opinion!)

Reply:
> But since this is a security feature of composer, I think we should limit the whitelist very strictly. You never know that somebody is able to sneak in an evil composer plugin with the magento vendor namespace (not necessarily from repo.magento.com or packagist.org), it's good to keep this list as strict as possible.
This is a good argument. 👍 

> (And on a personal note, I really would like to see Magento to stop creating extra composer plugins, look at the migration of composer v1 to composer v2, it was a disaster in Magento because of all the composer plugins it depends upon, this will happen again when composer v3 comes along one day, fewer plugins are better in my opinion!)
Yes, sure!